docs/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -29,7 +29,7 @@ Since version 1.12, Docker also provides `overlay2` storage driver which is much
 more efficient than `overlay` in terms of inode utilization. The `overlay2`
 driver is only compatible with Linux kernel 4.0 and later.
 
-For comparison between `overlay` vs `overlay2`, please also refer to [Select a
+For comparison between `overlay` vs. `overlay2`, please also refer to [Select a
```

Comment:
Thanks @thaJeztah 
The 'between and' is better, and I see 'please also refer to [Select a
storage driver](selectadriver.md#overlay-vs-overlay2)', so is it better that 'For comparison between `overlay` vs `overlay2`' is changed to 'For  `overlay` vs `overlay2`' ?